src/main.cpp

```diff
@@ -887,7 +887,6 @@ static bool run_dedicated_server(const GameParams &game_params, const Settings &
 			// Create server
 			Server server(game_params.world_path, game_params.game_spec,
 					false, bind_addr, true, &iface);
-			server.init();
 
 			g_term_console.setup(&iface, &kill, admin_nick);
```

Comment:
Are you sure g_term_console does not rely on `init()` being called before `setup()`?

Reply:
i will recheck, but i don't see any link between server and console setup

Reply:
the console uses only a run queue waiting events from other components, it's pretty isolated, no problem here